### PR TITLE
chore: remove redundant fully qualified PolicyService usage

### PR DESCRIPTION
**Pull request by hsiliev**
Use the same style across `ZaakafhandelParametersRESTService`
 
Solves: PZ-2931

### CODE REVIEW

**Review by felixcicatt — APPROVED** [review]

**Review by edgarvonk — APPROVED** [review]
nice. I think I also just changed these in my PR :-)

File: src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt [hunk 1]
```diff
@@ -78,7 +78,7 @@ class ZaakafhandelParametersRESTService @Inject constructor(
     @GET
     @Path("caseDefinition")
     fun listCaseDefinitions(): List<RESTCaseDefinition> {
-        PolicyService.assertPolicy(policyService.readOverigeRechten().beheren)
+        assertPolicy(policyService.readOverigeRechten().beheren)
```
Comment (bas-info-nl) [inline src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt:81]: Why not lift these assertions up into an annotation?

**Review by bas-info-nl — APPROVED** [review]

File: src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt (hunk 1 above)
Comment (hsiliev) [reply src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt:81]: Not a bad idea for the general use-case. @bas-info-nl would you create a story for that?

I think we misuse assertPolicy in a few places and throw all kinds of checks in, so we may still need the function.

Comment (edgarvonk) [reply src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt]: That would be nice yes but I think there may be some downsides to using annotations here like?
- will it not make unit testing harder?
- some policy checks need code to run first before they can be checked. These policies are not on the first line in the function/method in question

**Pull request merged**